source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -281,7 +281,19 @@ Connection Pool Behaviors
 Creating a Connection Pool
 --------------------------
 
-Pool creation is mostly an implementation detail specific to the driver language. If minPoolSize is set, the pool MUST immediately create enough connections such that totalConnections >= minPoolSize. These connections MUST be created in a non-blocking manner. Drivers SHOULD additionally connect these connections if it is possible to do so in a non-blocking manner, either via the use of a Background Thread or via asynchronous I/O.
+This specification does not define how a pool is to be created, leaving it
+up to the driver. Creation of a connection pool is generally an implementation
+detail of the driver, i.e., is not a part of the public API of the driver.
+
+The driver SHOULD only create connection pools for servers it has determined
```

Comment:
Does this belong in this spec? In my mind the CMAP spec describes a generic pool and its behaviors, not what types of server should have pools made for them. If we agree with this behavior, I think it should live in the SDAM spec

Reply:
Moved this section to the sdam spec.